apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -183,6 +185,28 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): J
         }
     }, [profileSchema, profileDetails.profileInfo]);
 
+    /**
+     * This will load the countries to the dropdown.
+     */
+    useEffect(() => {
+        const countryCodesToSkip = ["AQ", "BQ", "CW", "GG", "IM", "JE", "BL", "MF", "SX", "SS"];
```

Comment:
Is this needed since we have the util method? @DinikaSen

Reply:
This is for my account. Since the util method is in console, had to add this separately.

Reply:
Let's move it to common utils and use. Create an issue to track it.

Reply:
Created an issue to track moving this to common utils - https://github.com/wso2-enterprise/asgardeo-product/issues/3364

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/1972/commits/3a82b794fd61d8510b4f545ce44805b09f5681d4